src/Import/src/FMI3/fmi3_import_convenience.c

```diff
@@ -274,17 +252,21 @@ void fmi3_log_forwarding(fmi3_instance_environment_t inst, fmi3_status_t status,
     fmi3_import_t* fmu = (fmi3_import_t*)inst;
     jm_callbacks* cb;
     jm_log_level_enu_t logLevel;
+    const char* instanceName;
 
-    if(fmu) {
-         cb = fmu->callbacks;
-         buf = jm_vector_get_itemp(char)(&fmu->logMessageBufferCoded,0);
-    }
-    else  {
+    if (fmu) {
+        cb = fmu->callbacks;
+        buf = jm_vector_get_itemp(char)(&fmu->logMessageBufferCoded, 0);
+    } else {
+        // XXX: Why do even accept that fmu==NULL? What's the point of forwarding
+        // the logging to the FMU in that case? Also, it's not documented in the
+        // function's interface.
```

Comment:
I rewrote the function's documentation now... But the old one didn't mention it either, and it really doesn't make sense. We can probably remove it, but preferred to not mess with it right now.